solutions/devsprint-eduardo-bocato-1/SpaceApp/Resources/Assets.xcassets/badge.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
**[CHANGE REQUEST]**
if this is just for the previews, add it on PreviewAssets instead.